README.md

```diff
@@ -156,7 +156,27 @@ class CreateUser
 end
 ```
 
-You'll get an `ArgumentError` if you pass any params not defined in the `inputs_for` block, or if you don't supply all required params. This is completely opt-in so if you don't provide an `inputs_for` block, no checking is performed.
+### Specifying types and nullable fields
+
+You can now also optionally specify types and nullability for your inputs. For the `type:` option, any ruby class can be used.
+
+```ruby
+inputs_for(:call) do
+  required :name, type: String, null: false
```

Comment:
What about if we required a `null:` specification on all `required` fields? This would then be a RuntimeError because you're using the library incorrectly and would (hopefully) show up in tests before getting to production.